network/iptables.go

```diff
@@ -0,0 +1,9 @@
+package network
```

Comment:
No, that's not how we do interfaces in Go. The interface exists to abstract the behavior that your code is using from the backing implementation of that behavior _at your usage_ of that behavior. This package only needs _this subset_ of behavior, so the interface should be defined in this package.

Reply:
https://go.dev/doc/effective_go#interfaces
> "If something can do _this_, it can be used _here_."

Don't predefine an interface that simply matches a type.

Reply:
i guess iptables package used in other components like cns i guess.. if its just used in network package, then i agree